client/src/pages/dashboard/Dashboard.js

```diff
@@ -63,12 +73,15 @@ function Dashboard() {
   const [proposalListDefault, setProposalListDefault] = React.useState(proposals);
   const [proposalList, setProposalList] = React.useState(proposalHTML);
 
+  const [proposalTitle, setProposalTitle] = React.useState("Project Title");
+  const [proposalDescription, setProposalDescription] = React.useState("Lorem Ipsum fdsafdasfdasfas");
+
   /* Updates proposalList state based on SearchBar input. */
   const updateInput = (input) => {
     let filteredList = [];
     for (let i = 0; i < proposals.length; i++) {
       if (proposalListDefault[i].title.toLowerCase().includes(input.toLowerCase())) {
-        filteredList.push(<Row title={proposalListDefault[i].title} vote={proposalListDefault[i].voted ? proposalListDefault[i].voted : ""} />)
+        filteredList.push(<Row changeTitle={setProposalTitle} changeDescription={setProposalDescription} title={proposalListDefault[i].title} vote={proposalListDefault[i].voted ? proposalListDefault[i].voted : ""} />)
```

Comment:
There is a small bug where if you click on the same row twice, the page errors out. I haven't looked too deep into why this is happening, but it might be with what functions you are passing as callbacks when no change occurs.